.github/workflows/linux.yml

```diff
@@ -924,8 +924,10 @@ jobs:
         run: |
           python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
       - name: PyTorch Models Tests
+        # detectron2 requires torch to be alredy installed
         run: |
           python3 -m pip install -r ${{ env.MODEL_HUB_TESTS_INSTALL_DIR }}/torch_tests/requirements.txt
+          python3 -m pip install git+https://github.com/facebookresearch/detectron2.git 
```

Comment:
Did you consider adding this dependency to a requirement file? https://pip.pypa.io/en/stable/topics/vcs-support

Reply:
You can still create a separate requirement file for detectron (like `requirements_detectron.txt`). It may be useful for local runs, especially if more dependencies will be added or if we'll have to pin a version of this dependency due to some regression (people won't need to look up a working version in the pipeline)

Reply:
done